mappings/net/minecraft/world/chunk/ChunkBatcher.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS none/brk net/minecraft/world/chunk/ChunkBatcher
```

Comment:
If unsure, just get rid of the mapping for now. I've mapped some rendering stuff just now. Might help.